cartoframes/viz/kuviz.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, id, url, name, privacy=PRIVACY_PUBLIC):
 
     @classmethod
     def create(cls, html, name, context=None, password=None):
-        from cartoframes.auth import _default_context
+        from ..auth import _default_context
```

Comment:
Why do you prefer a relative path?

Reply:
I think it is a common practice to use relative imports in the project and absolute imports for your dependencies. It requires usually fewer chars and it decouples your implementation from the public API.

Reply:
Agree, I like relative imports for project module imports